tests/fakeprotocol_iot.py

```diff
@@ -466,7 +473,13 @@ def get_response_for_command(cmd):
 
         response = {}
         for target in request:
-            response.update(get_response_for_module(target))
+            if self.verbatim:
+                if target in self.proto:
+                    response.update({target: self.proto[target]})
+                else:
+                    response.update({"err_msg": "module not support"})
+            else:
+                response.update(get_response_for_module(target))
```

Comment:
Please no. We have to find a nicer solution for this as reading the if-if-else-jungle makes my head hurt already.

Reply:
Simplified